.github/workflows/integrate.yaml

```
@@ -130,7 +130,7 @@ jobs:
           provider: microk8s
           channel: ${{ matrix.microk8s-versions }}
           microk8s-addons: "dns hostpath-storage rbac metallb:10.64.140.43-10.64.140.49"
-          juju-channel: 3.1/stable
+          juju-channel: 3/stable
```

Can we keep this to `3.4` rather than `3`?  We have specific version that we're guaranteeing so testing against those feels more appropriate imo

```suggestion
          juju-channel: 3.4/stable
```